startupscript/dataproc/install-r.sh

```diff
@@ -0,0 +1,57 @@
+#!/bin/bash
+# install-r.sh
+# Installs the latest version of R
+#
+# This script is intended to be executed as a Dataproc cluster initialization
+# action so such that R is installed and configured on all cluster nodes during
+# creation and autoscaling.
+# For more information on Initialization actions, see:
+# https://cloud.google.com/dataproc/docs/concepts/configuring-clusters/init-actions
+
+set -o errexit
+set -o nounset
+set -o pipefail
+set -o xtrace
+
+# Emit a message with a timestamp
+function emit() {
+  echo "$(date '+%Y-%m-%d %H:%M:%S') $*"
+}
+readonly -f emit
+
+# Define directories and system files
+readonly LOGIN_USER="dataproc"
+readonly USER_HOME_DIR="/home/${LOGIN_USER}"
+readonly USER_WORKBENCH_CONFIG_DIR="${USER_HOME_DIR}/.workbench"
+readonly USER_BASHRC="${USER_HOME_DIR}/.bashrc"
+readonly R_BIN_DIR='/usr/lib/R/bin'
+readonly OUTPUT_FILE="${USER_WORKBENCH_CONFIG_DIR}/install-r-output.txt"
+
+# Send stdout and stderr from this script to a file for debugging.
+# Make the .wb directory as the user so that they own it and have correct linux permissions.
```

Comment:
The second line in this comment doesn't appear to be reflected in code.
(And the directory is `.workbench`, rather than `.wb`.)

Reply:
Ah, thank you